UnitsNet.Tests/CustomCode/LeakRateTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class LeakRateTests : LeakRateTestsBase
+    {
+        protected override double PascalQubicMeterPerSecondInOnePascalQubicMeterPerSecond => 1;
+
+        protected override double MillibarLiterPerSecondInOnePascalQubicMeterPerSecond => 10;
+
+        protected override bool SupportsSIUnitSystem => false;
+
+        protected override double TorrLiterPerSecondInOnePascalQubicMeterPerSecond => 7.5;
```

Comment:
This converter says `7.50061683`?
https://www.sensorsone.com/leak-rate-converter/

Reply:
This one says 7.5
https://www.helium-leak.eu/calculations-and-units/

Reply:
7.5 here too
https://www.leybold.com/en-in/knowledge/vacuum-fundamentals/leak-detection/definition-and-measurement-of-vacuum-leaks